plugins/woocommerce-admin/client/products/sections/product-inventory-section.tsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Card, CardBody, TextControl } from '@wordpress/components';
+import { getAdminLink } from '@woocommerce/settings';
+import { Link, useFormContext } from '@woocommerce/components';
+import { Product } from '@woocommerce/data';
+import { recordEvent } from '@woocommerce/tracks';
+
+/**
+ * Internal dependencies
+ */
+import { getTextControlProps } from './utils';
+import { ProductSectionLayout } from '../layout/product-section-layout';
+
+export const ProductInventorySection: React.FC = () => {
+	const { getInputProps } = useFormContext< Product >();
+
+	return (
+		<ProductSectionLayout
+			title={ __( 'Inventory', 'woocommerce' ) }
+			description={
+				<>
+					<span>
+						{ __(
+							'Set up and manage inventory for this product, including status and available quantity.',
+							'woocommerce'
+						) }
+					</span>
+					<Link
+						href={ getAdminLink(
+							'admin.php?page=wc-settings&tab=products&section=inventory'
+						) }
+						target="_blank"
+						type="wp-admin"
+						onClick={ () => {
+							recordEvent( 'add_product_inventory_help' );
```

Comment:
I think the Tracks event should be something like `product_editor_inventory_help_clicked`.

* `add_product` isn't accurate -- this will also be fired when editing a product
* There is no action

(naming guidelines, specifying that an action should be in the name: PCYsg-4S2-p2)

Reply:
Agreed.  I was following the existing naming conventions that were already in place for the product experience.  Is there a sheet somewhere specifying those names or were these created on the spot?  cc @pmcpinto @octaedro

Reply:
I don't love clean up issues, but perhaps it would be best to address Tracks naming across the board in a follow up clean up issue, so we can get this functionality merged to unblock further MVP work.

As you mention, you were just following existing naming conventions on the new product experience. And, those existing conventions are not in line with best practices/guidelines. /cc @AnnaMag @pmcpinto

Reply:
I do not see any existing wcadmin events that could be directly copied. I believe the pattern could be: `product_inventory_manage_settings_clicked`
(e.g. following the mobile event `product_detail_view_inventory_settings_tapped`).